setup.py

```diff
@@ -5,6 +5,6 @@
       author="Superbio AI",
       author_email="smorgan@superbio.ai",
       url='https://github.com/Superbio-ai/app-sbioutils',
-      install_requires=['requests==2.22.0', 'boto3==1.21.27','pandas==1.2.5', 'anndata==0.8.0', 'scanpy==1.9.1', 'numpy==1.21', 'yaml'],
-      packages=['sbioapputils']
+      install_requires=['requests==2.22.0', 'boto3==1.21.27', 'pyflakes', 'pycodestyle', 'python-dotenv'],
```

Comment:
same comment: we removed pandas and anndata and scanpy and numpy but they are needed for file-process, does it still work?